From 225a4de0d9d1a2a4d55c476a46e85fffa8c3c5d8 Mon Sep 17 00:00:00 2001
From: Dmitry Eremin-Solenikov <dbaryshkov@gmail.com>
Date: Mon, 27 Jan 2014 16:57:48 +0400
Subject: [PATCH 5/7] ARM: pxa: tosa: fix several build failures

After recent refactoring of <mach/gpio.h> several tosa drivers fail to
build thanks to PXA_NR_BUILTIN_GPIO symbol being undefined. Include
<mach/irqs.h> into <mach/tosa.h> to make that symbol available where
required.

Signed-off-by: Dmitry Eremin-Solenikov <dbaryshkov@gmail.com>
---
 arch/arm/mach-pxa/include/mach/tosa.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/mach-pxa/include/mach/tosa.h b/arch/arm/mach-pxa/include/mach/tosa.h
index 2bb0e86..46e525bf 100644
--- a/arch/arm/mach-pxa/include/mach/tosa.h
+++ b/arch/arm/mach-pxa/include/mach/tosa.h
@@ -13,6 +13,9 @@
 #ifndef _ASM_ARCH_TOSA_H_
 #define _ASM_ARCH_TOSA_H_ 1
 
+/* for PXA_NR_BUILTIN_GPIO */
+#include <mach/irqs.h>
+
 /*  TOSA Chip selects  */
 #define TOSA_LCDC_PHYS		PXA_CS4_PHYS
 /* Internel Scoop */
-- 
1.8.5.2

